src/util.h

```diff
@@ -18,6 +18,7 @@
 #ifdef _WIN32
 #include "win32port.h"
 #else
+#include <inttypes.h>
```

Comment:
`win32port.h` defines `int64_t`, etc., and related printf-specifiers. This header defines the print specifiers on non-Windows. However, based on the CI failing, I probably need to define something to enable those on older systems, or since this is C++ code, maybe try `cinttypes`.